iot-hub/Samples/device/PnpDeviceSamples/Thermostat/Thermostat.csproj

```diff
@@ -19,6 +19,10 @@
     <PackageReference Include="DotNetty.Common" Version="0.7.1" />
   </ItemGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\PnpConvention\PnpHelpers.csproj" />
```

Comment:
For some historical context, the Thermostat sample using the existing APIs with PnP was designed to get those users onboarded who didn't want to add additional helpers to their code or use any new APIs. For this reason, the helper was deliberately left out of this particular sample. I know you've added it in to get the writable response helper, but I think it'll fit well with the rest of the sample to leave the helpers out and format the payload ourselves.

Reply:
I see, changed it back for Thermostat sample.